bloom/block_amd64.s

```diff
@@ -0,0 +1,70 @@
+//go:build !purego
+
+#include "textflag.h"
+
+DATA ones+0(SB)/4, $1
+DATA ones+4(SB)/4, $1
+DATA ones+8(SB)/4, $1
+DATA ones+12(SB)/4, $1
+DATA ones+16(SB)/4, $1
+DATA ones+20(SB)/4, $1
+DATA ones+24(SB)/4, $1
+DATA ones+28(SB)/4, $1
+GLOBL ones(SB), RODATA|NOPTR, $32
+
+DATA salt+0(SB)/4, $0x47b6137b
+DATA salt+4(SB)/4, $0x44974d91
+DATA salt+8(SB)/4, $0x8824ad5b
+DATA salt+12(SB)/4, $0xa2b7289d
+DATA salt+16(SB)/4, $0x705495c7
+DATA salt+20(SB)/4, $0x2df1424b
+DATA salt+24(SB)/4, $0x9efc4947
+DATA salt+28(SB)/4, $0x5c6bfb31
+GLOBL salt(SB), RODATA|NOPTR, $32
+
+// This initial block is a SIMD implementation of the mask function declared in
```

Comment:
Did you write this yourself or borrow it from somewhere? If borrowed could we link to the original?

Reply:
I wrote it.